src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,26 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Locale;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 public class JavaDateTimeApi {
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
+    private static final ZoneOffset UA_TIME_ZONE = ZoneOffset.of("+02:00");
+    private static final String FORMATTERS  = "dd MMMM y HH:mm";
```

Comment:
let's make formatter as a constant not only string

Reply:
done